content/blog/2022-06-08-announcing-localstack-discussion-pages/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Announcing LocalStack Discussion Pages
+description: "Introducing, LocalStack's Community Discussion Pages. Get started with conversations around feedback/bugs/suggestions for the LocalStack as a product and platform!"
+lead: "Introducing, LocalStack's Community Discussion Pages. Get started with conversations around feedback/bugs/suggestions for the LocalStack as a product and platform!"
+date: 2022-06-08T12:51:11+05:30
+lastmod: 2022-06-08T12:51:11+05:30
+images: []
+contributors: ["Harsh Mishra"]
+tags: ['news']
+---
+
+With the rise of the LocalStack Community on GitHub, Slack, and Social-media platforms like LinkedIn and Twitter, we have received a lot of feedback from our users. These feedbacks range from new feature requests for our LocalStack Team and Web Application platform, bug reports, enhancements, and further additions from various developers distributed across our user base. 
+
+We have been handling these requests through our GitHub Issue tracker, which has been limited to our open-sourced cloud emulation and doesn't cover our premium features. To address this and collaborate with our community better, we announce the all-new LocalStack Discussion Pages. 
+
+LocalStack Discuss form is also a positive step in engaging with our community members and listening to their feedback. While our Slack community is limited to taking immediate bug reports and feature requests and working with our community from a close quarter, our Discussion Pages will help developers to give us feedback, join the discussions on our core emulation framework, Team & Enterprise product, and future direction of LocalStack as a company.
+
+## Being a part of LocalStack Discuss
+
+The LocalStack Discussion Pages will be the dedicated place to discuss topics and questions related to:
+
+- Discussions and announcements about LocalStack Pro and Community releases.
+- LocalStack Documentation, reference guides, case studies, and more.
+- Useful LocalStack tips, tricks, and best practices for developing your cloud & serverless applications locally.
+- Contributing to the LocalStack Community codebase and steering the development of our open-source projects.
+- Send constructive feedback about LocalStack as a product and as a company and how we can do better!
+
+However, it will not be a place for ad-hoc reports or support questions; Please connect with our support team on [Slack](https://localstack.cloud/contact) or create a GitHub issue on our open-source [GitHub Issue tracker](https://github.com/localstack/localstack/issues) for our help with your issues.
```

Comment:
Create a better strategy for users to connect with us, either via GitHub or Slack.

Reply:
Suggestions from @thrau :

- Discourse can be used for support and to ask questions. 
- Common issues should be indexed and identified by normal users.
- Rephrase: `If you are looking for real-time support from LocalStack Team and Community, please use Slack. For bug reports please use the GitHub Issue Tracker.`